cfme/ansible/repositories.py

```diff
@@ -110,6 +117,15 @@ def db_object(self):
     def playbooks(self):
         return self.collections.playbooks
 
+    def get_detail(self, title, field, refresh=False):
+        view = navigate_to(self, "Details")
+        if refresh:
+            if self.appliance.version < "5.9":
+                view.browser.refresh()
+            else:
+                view.refresh.click()
+        return getattr(view.entities, title.lower().replace(" ", "_")).get_text_of(field)
```

Comment:
Same issue as mentioned by @mshriver in #6157 but I'd let it slide and we can fix all the instances in the next sprint.